src/nvim/optionstr.c

```diff
@@ -2005,6 +2005,15 @@ const char *did_set_winhighlight(optset_T *args)
   return NULL;
 }
 
+/// The 'winborder' option is changed.
+const char *did_set_winborder(optset_T *args)
+{
+  if (opt_strings_flags(p_winbd, opt_winborder_values, NULL, true) != OK) {
```

Comment:
Why is the last argument `true`? This is not a list option.

Actually the `did_set_winborder()` function is not needed anymore. `did_set_str_generic()` is automatically used when `values` is present.

Reply:
Fix in #32978